### PR TITLE
Funds could be stuck if IRM reverts

### PR DESCRIPTION
**Pull request by MerlinEgalite**
Fixes https://github.com/cantinasec/review-morpho-blue-1/issues/41

### CODE REVIEW

File: src/interfaces/IMorpho.sol [hunk 1]
```diff
@@ -143,7 +143,7 @@ interface IMorpho {
     /// - The token can revert on `transfer` and `transferFrom` for a reason other than an approval or balance issue.
     /// - A very high amount of assets (~1e35) supplied or borrowed can make the computation of `toSharesUp` and
     /// `toSharesDown` overflow.
-    /// - The IRM can revert on `borrowRate`.
+    /// - The IRM can revert on `borrowRate` which means it could lead to funds being stuck.
```
Comment (Rubilmax) [inline src/interfaces/IMorpho.sol:146]: I believe it is the case for all of these liveness-breaking situations, so we should duplicate this comment everytime?

But I understand that "break Morpho's liveness properties" imply that funds may get stuck in some cases, so I would say this comment is simply highlighting that no liveness => funds may be stuck
Perhaps we want to change the wording of the sentence introducing the bullet points?

Comment (MerlinEgalite) [reply src/interfaces/IMorpho.sol:146]: That was an argument made in the related issue. We can acknowledge it also

Comment (QGarchery) [reply src/interfaces/IMorpho.sol:146]: Agree with @Rubilmax, I would just add that funds **could** get stuck after "break Morpho's liveness properties"

Comment (Rubilmax) [reply src/interfaces/IMorpho.sol:146]: Ah, right: https://github.com/cantinasec/review-morpho-blue-1/issues/41#issuecomment-1780862265
I'm fine with both!

**Review by Rubilmax — APPROVED** [review]

**Review by QGarchery — APPROVED** [review]

**Review by Jean-Grimal — DISMISSED** [review]

**Review by MathisGD — APPROVED** [review]

**Review by QGarchery — APPROVED** [review]

**Pull request merged**